src/defs/server.ts

```diff
@@ -1265,98 +1265,756 @@ tmpDef.argDescs = {};
-- textDestroyDisplay => Missing
-- textDisplayAddObserver => Missing
-- textDisplayAddText => Missing
-- textDisplayRemoveObserver => Missing
-- textDisplayRemoveText => Missing
-- textDisplayGetObservers => Missing
-- textDisplayIsObserver => Missing
-- iprint => Missing
-- inspect => Missing
-- isOOPEnabled => Missing
-- getVehiclesOfType => Missing
-- getModelHandling => Missing
-- setModelHandling => Missing
-- setVehicleIdleRespawnDelay => Missing
-- setVehicleRespawnDelay => Missing
-- setVehicleRespawnPosition => Missing
-- setVehicleHandling => Missing
-- setVehicleVariant => Missing
-- getSkyGradient => Missing
-*/
\ No newline at end of file
+tmpDef = new MTAFunction;
+tmpDef.label = "aclGet";
+tmpDef.description = " Get the ACL with the given name. If need to get most of the ACL's, you should consider using aclList to get a table of them all.";
+tmpDef.returnType = "acl";
+tmpDef.args = ["string aclName"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclCreate";
+tmpDef.description = " This function creates an ACL entry in the Access Control List system with the specified name.";
+tmpDef.returnType = "acl";
+tmpDef.args = ["string aclName"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclDestroy";
+tmpDef.description = " This function destroys the ACL passed. The destroyed ACL will no longer be valid.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["acl theACL"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclGetName";
+tmpDef.description = " Get the name of given ACL.";
+tmpDef.returnType = "string";
+tmpDef.args = ["acl theAcl"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclGetRight";
+tmpDef.description = " This function returns whether the access for the given right is set to true or false in the ACL.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["acl theAcl, string rightName"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclSetRight";
+tmpDef.description = " This functions changes or adds the given right in the given ACL. The access can be true or false and specifies whether the ACL gives access to the right or not.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["acl theAcl, string rightName, bool hasAccess"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Server;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "aclListRights";
+tmpDef.description = " This function returns a table of all the rights that a given ACL has.";
+tmpDef.returnType = "table";
+tmpDef.args = ["acl theACL, string allowedType"];
```

Comment:
args is an array and every arg should be one element in the array. Please change it everywhere when there are multiple arguments :/
```ts
["ACL theAcl", "string allowedType"]
```